### PR TITLE
Remove global utils file

### PR DESCRIPTION
**Pull request by jitinmaher**
# Title

Remove global utils file and use jest globals instead for mocking

## Description of what's changing

Remove global utils file and use jest globals instead for mocking

## What else might be impacted?

`src/js/common/browserInfoHelper.js`  is directly consuming the global object now

## Which issue does this PR relate to?

https://github.com/intuit/user-data-for-fraud-prevention/issues/34

## Checklist

[ ] Tests are written and maintain or improve code coverage
[ ] I've tested this in my application using `yarn link` (if applicable)
[ ] You consent to and are confident this change can be released with no regression

### CODE REVIEW

**Comment by jitinmaher** [comment]
@skodamarthi fixed broken tests from previous PR's

Screenshot 2020-10-07 at 11 39 55 PM

File: tests/unit/common/browserInfoHelper.test.js [hunk 1]
```diff
@@ -208,27 +219,30 @@ describe("BrowserInfoHelper", () => {
...
+
+    global.RTCPeerConnection = MockRTCPeerConnection;
+    
+    screenSpy.mockImplementation(() => ({
       width: 1019,
       height: 1021,
-      colorDepth: 17,
-    });
+      colorDepth: 17
+    }))
```
Comment (skodamarthi) [inline tests/unit/common/browserInfoHelper.test.js:245]: Minor: Please add the semi colon at the end to follow the same pattern everywhere.

File: tests/unit/common/browserInfoHelper.test.js [hunk 2]
```diff
@@ -250,11 +264,11 @@ describe("BrowserInfoHelper", () => {
...
 
   it("getScreen", async () => {
-    jest.spyOn(globalsUtil, "getScreen").mockReturnValue({
+    screenSpy.mockImplementation(() => ({
       width: "900px",
       height: 1021,
-      colorDepth: 17,
-    });
+      colorDepth: 17
+    }))
```
Comment (skodamarthi) [inline tests/unit/common/browserInfoHelper.test.js:271]: Minor: Please add the semi colon at the end to follow the same pattern everywhere.

**Comment by jitinmaher** [comment]
@skodamarthi updated with comma and semi

**Review by skodamarthi — APPROVED** [review]

**Pull request merged**